### ISSUE TITLE
H3 py conda forge packaging for apple silicon

### ISSUE THREAD

**Issue by geoHeil** [uber/h3-py#311]
Hi,

I am trying to fixup the conda-forge build of H3-py https://github.com/conda-forge/h3-py-feedstock/pull/34

but feel a bit stuck.

From https://github.com/uber/h3-py/issues/204 it looks like the build of pip install h3 should already support apple silicon. However, it looks like whatever is provided as H3 from conda does not.

At least I am stuck with:

```
#ERROR: Failed building wheel for h3
  Building wheel for h3 (pyproject.toml): finished with status 'error'
Failed to build h3
ERROR: Could not build wheels for h3, which is required to install pyproject.toml-based projects
```

Can someone here helped me fix the apple silicon build of H3-py?

**Comment by nelsonauner** [comment]
Correct, I believe the issue is that conda-forge only has version 3.7.4 (https://anaconda.org/conda-forge/h3-py/files) whereas pip supports 3.76 (https://pypi.org/project/h3/) 

Can you try installing from pip? Like you, I also use conda, but just installed this one as a one-off via pip
